.gitignore

```diff
@@ -1,3 +1,3 @@
-/target
+target/
```

Comment:
Since this is generated by `cargo` itself I don't think this change adds anything

Reply:
In contrast, it is vital for portability.  On a Unix machine, there are absolutely different meanings.

`target/` is what you want:  a subdirectory of the repository root which is in the scope of the Git repository.

`/target` is something totally different:  a subdirectory of the **machine's** root which does not exist (usually, at least).  Both Git and the operating system might get confused by that which implies side effects rather to prevent than to encourage.

Reply:
I just used the `cargo new project` command on linux and it created the exact same `.gitignore` file and `git` itself was fine with it.

So again, this change does not add anything.

Reply:
Yes, you are right.  I just tested with the original.

I reverted the change with the latest commit.  In order to skip the corresponding commits, I, again, recommend squashing as the strategy of choice for the merge which would also solve the commit message issue.